packages/apps/api-authorizer/src/authorization.ts

```diff
@@ -12,7 +12,9 @@ type HttpVerb =
   | 'CONNECT';
 
 type AccessControlRule = (
-  jwt: Jwt,
+  jwt: Jwt & {
+    payload: (JwtPayload & { gty?: string }) | string
```

Comment:
I don't think we need both checks and `gty` seems more hacky that the `@client` suffix (which is, at least, inside a meaningful claim) - can we just use the latter?

Reply:
Sure, changed.